packages/ra-language-english/src/index.ts

```diff
@@ -111,9 +111,15 @@ const englishMessages: TranslationMessages = {
             page_out_from_end: 'Cannot go after last page',
             page_out_from_begin: 'Cannot go before page 1',
             page_range_info: '%{offsetBegin}-%{offsetEnd} of %{total}',
+            partial_page_range_info:
+                '%{offsetBegin}-%{offsetEnd} of more than %{offsetEnd}',
+            current_page: 'Page %{page}',
+            page: 'Go to page %{page}',
+            first: 'Go to first page',
+            last: 'Go to last page',
+            next: 'Go to next page',
+            previous: 'Go to previous page',
```

Comment:
BC detected: You change `prev` for `previous`

Reply:
indeed, I need to add an entry to the Upgrade guide